plugins/ICGenJetProducer.cc

```diff
@@ -61,7 +62,8 @@ void ICGenJetProducer::produce(edm::Event& event,
     dest.set_energy(src.energy());
     dest.set_charge(src.charge());
     dest.set_flavour(0);
-    dest.set_n_constituents(src.getGenConstituents().size());
+    if(is_slimmed_) dest.set_n_constituents(src.numberOfDaughters());
+    else dest.set_n_constituents(src.getGenConstituents().size());
```

Comment:
I think they would give the same answer; numberOfDaughters() points to the PackedGenParticles collection, which contains all stable gen particles.
The standard way to request constituents won't work. Links to the daughters in the PackedGenParticles collection are available and there are links between the packed and pruned gen particles, so we can do something (but maybe not as much as for normal gen jets)